admin_ui/src/utils.ts

```diff
@@ -140,6 +145,10 @@ export function convertFormValue(params: {
     if (value == "null") {
         value = null
     } else if (property.extra?.nullable && value == "") {
+        // TODO We can potentially remove this in the future - isNullable does
+        // what we need.
+        value = null
+    } else if (isNullable(property) && value == "") {
```

Comment:
This is problematic with the current version of Piccolo - in the Pydantic model for tables, they default to optional, unless `required=True`.

This PR fixes it:

https://github.com/piccolo-orm/piccolo/pull/1141

But need to decide whether it's a good idea or not.

Reply:
I don't really understand what the implication here is, maybe @sinisaos has some thoughts?

Reply:
Thanks both for testing it. 

I'll change this slightly, so it's completely backwards compatible - i.e. the current behaviour for tables, and for custom forms it uses `isNullable`.